src/plugins/buflist/buflist-config.c

```diff
@@ -773,6 +776,30 @@ buflist_config_init ()
         NULL, NULL, NULL,
         &buflist_config_change_buflist, NULL, NULL,
         NULL, NULL, NULL);
+    buflist_config_format_tls_version = weechat_config_new_option (
```

Comment:
Currently, the name used in WeeChat is "ssl" everywhere in options, but I agree migrating to "tls" is more meaningful as we don't use SSL any more.
Renaming the historical options is just annoying for people upgrading to newer versions, as the server options "ssl*" would be renamed and old values would be lost.
So we can keep this anyway, even if not consistent with all options.